<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestoreDialog</class>
 <widget class="QDialog" name="RestoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title: window">Restore Guide</string>
  </property>
  <layout class="QVBoxLayout" name="mTopLevelVLayout">
   <item>
    <widget class="QStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mRestoreLocationPage">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mRestoreOriginalButton">
           <property name="text">
            <string comment="@action:button">Restore to original location</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mRestoreCustomButton">
           <property name="text">
            <string comment="@action:button">Choose where to restore</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mDestinationPage">
      <layout class="QVBoxLayout" name="mDestinationVLayout">
       <item>
        <layout class="QHBoxLayout" name="mDestinationHLayout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mDestBackButton">
           <property name="text">
            <string comment="@action:button">Back</string>
           </property>
           <property name="icon">
            <iconset theme="arrow-left">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mDestNextButton">
           <property name="text">
            <string comment="@action:button">Next</string>
           </property>
           <property name="icon">
            <iconset theme="arrow-right">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mConfirmOverwritePage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="mConflictTitleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mNewNameRadioButton">
         <property name="text">
          <string comment="@option:radio ">Restore the folder under a new name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="mNewFolderNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="mOverwriteRadioButton">
         <property name="text">
          <string comment="@option:radio">Merge folders</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="mOverwriteListboxLayout">
           <item>
            <widget class="QLabel" name="mConfirmOverwriteLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string comment="@info Question to user, in dialog">The following files would be overwritten, please confirm that you wish to continue.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="mFileConflictList">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mOverwriteBackButton">
           <property name="text">
            <string comment="@action:button go to previous page in dialog">Back</string>
           </property>
           <property name="icon">
            <iconset theme="arrow-left">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mConfirmButton">
           <property name="text">
            <string comment="@action:button User answers &quot;yes I confirm: overwrite files&quot;">Confirm</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-ok-apply">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mRestorationProgressPage">
      <layout class="QVBoxLayout" name="mRestorationVLayout">
       <item>
        <widget class="QStackedWidget" name="mRestorationStackWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="mRestoreProgressLayout">
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="mErrorLabel">
             <property name="text">
              <string comment="@label above the detailed error messsage">An error occured while restoring:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="mRestorationOutput">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>149</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string comment="@label">Restoration completed sucessfully!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>192</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="KPushButton" name="mOpenDestinationButton">
               <property name="text">
                <string>Open Destination</string>
               </property>
               <property name="icon">
                <iconset theme="folder">
                 <normaloff/>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>192</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>149</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mCloseButton">
           <property name="text">
            <string comment="@action:button">Close</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-close">
             <normaloff/>
            </iconset>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mRestoreOriginalButton</tabstop>
  <tabstop>mRestoreCustomButton</tabstop>
  <tabstop>mDestNextButton</tabstop>
  <tabstop>mOverwriteBackButton</tabstop>
  <tabstop>mDestBackButton</tabstop>
  <tabstop>mFileConflictList</tabstop>
  <tabstop>mConfirmButton</tabstop>
  <tabstop>mRestorationOutput</tabstop>
  <tabstop>mCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>RestoreDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOverwriteRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFileConflictList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOverwriteRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mConfirmOverwriteLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNewNameRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mNewFolderNameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
